tests/components/mqtt/test_init.py

```diff
@@ -118,6 +118,20 @@ def test_service_call_with_payload_doesnt_render_template(self):
         }, blocking=True)
         self.assertFalse(mqtt.MQTT_CLIENT.publish.called)
 
+    def test_service_call_without_payload_or_payload_template(self):
```

Comment:
This comment is attached to a line. You have added a removed test back which was removed because it was impossible to happen.